frontend/public/components/storage/create-pvc.tsx

```diff
@@ -266,16 +282,22 @@ export const CreatePVCForm: React.FC<CreatePVCFormProps> = (props) => {
         Volume Mode
       </label>
       <div className="form-group">
-        {volumeModeRadios.map((radio) => (
-          <RadioInput
-            {...radio}
-            key={radio.value}
-            onChange={handleVolumeMode}
-            inline
-            checked={radio.value === volumeMode}
-            name="volumeMode"
-          />
-        ))}
+        {storageProvisioner.includes(cephStorageProvisioners[0]) ? (
+          <div>
+            Filesystem <InfoToolTip />
+          </div>
```

Comment:
you dont have to wrap it in div
```suggestion
          <>
            Filesystem <InfoToolTip />
          </>
```

Reply:
fixed